### PR TITLE
Update Playwright projects

### PR DESCRIPTION
**Pull request by teenoh**
This PR updates existing playwright projects to use different browser setup.

### CODE REVIEW

File: site/playwright.config.ts [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
+  "iPhone 11 Pro",
+  "Pixel 5",
+];
+
 const config: PlaywrightTestConfig = {
   forbidOnly: ci,
   projects: [
-    {
+    ...devicesToUse.map((deviceName) => ({
       name: "integration",
```
Comment (kachkaev) [inline site/playwright.config.ts:18]: Looks like we end up with multiple projects having the same name 🤔 Does Playwright support this?

Comment (teenoh) [reply site/playwright.config.ts:18]: It appears they do, or at least that's the recommended approach [here](https://github.com/microsoft/playwright/issues/13046#issuecomment-1078259500)

File: site/tests/universal/page-header.test.ts [hunk 2]
```diff
@@ -12,19 +36,32 @@ test("page header navigation works", async ({ page }) => {
...
   await page.locator("text=Close").click();
   await expect(page).toHaveURL("/docs");
   await expect(page.locator('h1:has-text("Introduction")')).toBeVisible();
 
-  await page.locator("header >> text=Sign Up").click();
+  if (isMobile) {
+    await openMobileNav(page);
+  }
+
+  // @todo: fix error mobile nav isn't visible at this point
```
Comment (teenoh) [inline site/tests/universal/page-header.test.ts:63]: Skipped this test on mobile for now. Had a weird bug where the mobile nav isn't visible at this point. Will address in a follow up

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -216,7 +216,7 @@ jobs:
 
       - name: Run Playwright tests
         if: ${{ success() || failure() }}
-        run: yarn workspace @blockprotocol/site playwright test --project=integration
+        run: yarn workspace @blockprotocol/site playwright test --project=integration-chrome --project=integration-firefox --project=integration-safari --project=integration-iphone --project=integration-pixel
```
Comment (kachkaev) [inline .github/workflows/ci.yml:219]: It'll be nice to use wildcard project name once this is supported upstream: https://github.com/microsoft/playwright/issues/15128

File: site/tests/universal/homepage.test.ts [hunk 4]
```diff
@@ -45,21 +45,23 @@ test("Home page should contain key elements", async ({ page }) => {
     await page.locator('[data-testid="block-slider"] >> .slick-slide').count(),
   ).toBeGreaterThan(5);
 
-  const finalCTA = page.locator('[data-testid="final-cta"]');
+  if (!isMobile) {
+    const finalCTA = page.locator('[data-testid="final-cta"]');
 
-  // @todo: Add tests to handle authenticated users, they shouldn't see this section
-  await expect(finalCTA).toBeVisible();
+    // @todo: Add tests to handle authenticated users, they shouldn't see this section
```
Comment (kachkaev) [inline site/tests/universal/homepage.test.ts:51]: I suggest handing authenticated users in another test file. We can keep this one scoped to guests. Too  many `if`s can make tests fragile and complex.

File: site/tests/universal/page-header.test.ts [hunk 5]
```diff
@@ -1,9 +1,33 @@
...
+    page.locator("[data-testid='mobile-nav-trigger']"),
+  ).toBeVisible();
+  await page.locator("[data-testid='mobile-nav-trigger']").click();
+  await expect(page.locator("[data-testid='mobile-nav']")).toBeVisible();
+};
+
+test("page header navigation works", async ({ page, isMobile }) => {
+  if (isMobile) {
+    test.skip();
+  }
```
Comment (kachkaev) [inline site/tests/universal/page-header.test.ts:18]: I assume that this is a temporary statement because there is a ternary expression below

**Review by kachkaev — APPROVED** [review]
🎭 FTW!

File: site/tests/universal/page-header.test.ts (hunk 5 above)
Comment (teenoh) [reply site/tests/universal/page-header.test.ts:18]: Yes... That's what I was referring to here => https://github.com/blockprotocol/blockprotocol/pull/444#discussion_r924520450

File: site/tests/universal/homepage.test.ts (hunk 4 above)
Comment (teenoh) [reply site/tests/universal/homepage.test.ts:51]: I agree.. I intended to write the flow for authenticated users in another `test(...)` in the same file. Happy to move to another test file when I implement

File: .github/workflows/ci.yml (hunk 3 above)
Comment (teenoh) [reply .github/workflows/ci.yml:219]: Nice!

**Pull request merged**